### ISSUE TITLE
Make achieved unlocks visible in the server

### ISSUE THREAD

**Issue by Shepard** [Shepard/bardbot#5]
Depends on #2.

### Motivation

With the basic unlock system, the player might be unfirmed that they achieved an unlock. With #3 an unlock can be used to suggest more (potentially secret) stories conditionally and using #4 an unlock in one story can influence the behaviour of another story.
However, the information that a player has achieved an unlock is not available to anyone outside of the playthrough.

### Solution

Potential scenarious are:

- The story owner or an admin gets informed via DM that a player triggered an unlock. They could react to this by awarding the player with a role in the server.
- An admin might want to check which players have achieved a certain unlock or which unlocks have been achieved by a certain player. Again, they could use this to hand out some kind of reward later on. Possible command: `/manages-stories show-unlocks <user>`.
- An unlock could be posted as an achievement in the server in a designated channel.
- Unlocks could be tracked as stats. ("14 other people have unlocked this." / "15% of players have unlocked this.")

Feedback needs to be collected which of these scenarios are relevant to server admins.
Potential privacy concerns need to be considered.
